docs/source/remote-data-services.rst

```diff
@@ -187,7 +187,13 @@ The following parameters may be passed to s3fs using ``storage_options``:
     - use_ssl: Whether connections are encrypted and secure (default True)
 
     - client_kwargs: Dict passed to the `boto3 client`_, with keys such
-      as `region_name` or `endpoint_url`
+      as `region_name` or `endpoint_url`. Notice: do not pass the `config`
+      option here, which will cause an ``TypeError: client() got multiple
+      values for keyword argument 'config'``, pass it's content to
+      `config_kwargs` instead.
+
+    - config_kwargs: Dict passed to the `s3fs python client`_, which act as
+      the `boto3 client's config`_ option.
```

Comment:
The text here is not clear. I would say passed to the `s3fs.S3FileSystem` constructor, which passes is it as the `config=` keyword to the `boto3 client`